Add new section for AnnData in Changelog

allyhawkins opened the pull request:
Closes #189 

Here, I created a new entry in the Changelog that includes everything relevant to AnnData additions. I added a placeholder for the heading that will need to be replaced with the date of release when the time comes. 

- Are there any items that you think are not relevant to be in the Changelog entry? 
- Are there any items that should be there that are missing? (I didn't include mention of the clustering assignments, but I could)

jaclyn-taroni commented on docs/CHANGELOG.md:
```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
+* All samples now have an assigned `participant_id`, which can be found in `single_cell_metadata.tsv`.
+* All data files now include both the gene expression data and metadata for each sample (e.g., age, sex, organism, ethnicity, diagnosis, and tissue location).
+For more information on the contents of the data files, see {ref}`the Single-cell gene expression file contents page<sce_file_contents:Single-cell gene expression file contents>`.
+* If provided by the original submitter, data files will also include cell type annotations.
```
```suggestion
* Data files will include cell type annotations provided by submitters when applicable.
```

jaclyn-taroni commented on docs/CHANGELOG.md:
```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 
+## PLACEHOLDER FOR RELEASE DATE
+
+* The data for all samples, other than multiplexed samples, can now be downloaded as either a `SingleCellExperiment(R)` or `AnnData(Python)` object contained in a RDS or HDF5 file, respectively.
```
Is there a section we should link to after the first section as well?

```suggestion
* Downloads for most projects are now available in [`AnnData`](https://anndata.readthedocs.io/en/latest/index.html) format as HDF5 files.
Multiplexed samples are not yet supported.
```

I don't think we need to state that the `SingleCellExperiment` downloads are available.

jaclyn-taroni commented on docs/CHANGELOG.md:
```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
+
+* The data for all samples, other than multiplexed samples, can now be downloaded as either a `SingleCellExperiment(R)` or `AnnData(Python)` object contained in a RDS or HDF5 file, respectively.
+* The sample metadata found in `single_cell_metadata.tsv` has been updated to include ontology term ids for age, sex, organism, ethnicity, diagnosis, and tissue location, when available.
+* All samples now have an assigned `participant_id`, which can be found in `single_cell_metadata.tsv`.
```
I think it'd be helpful to give more information about what we _used to do_ to provide more context for the potential impact:
```suggestion
* All samples now have an assigned `participant_id`, which can be found in `single_cell_metadata.tsv`.
Previously, `participant_id` were only assigned when multiple samples mapped to the same participant for most projects.
```

jaclyn-taroni commented on docs/CHANGELOG.md:
```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
+## PLACEHOLDER FOR RELEASE DATE
+
+* The data for all samples, other than multiplexed samples, can now be downloaded as either a `SingleCellExperiment(R)` or `AnnData(Python)` object contained in a RDS or HDF5 file, respectively.
+* The sample metadata found in `single_cell_metadata.tsv` has been updated to include ontology term ids for age, sex, organism, ethnicity, diagnosis, and tissue location, when available.
```
Is there a section for each download type we could link to here?

allyhawkins replied:
This is referring to the metadata table that's included in the download, not the metadata that's in the actual object. I added a link to that section of the docs.

jaclyn-taroni reviewed (commented):
Thanks for getting this going! My general comment is that a few of these points would benefit from some links to sections of the docs when possible.

allyhawkins commented:
Thanks for looking at this @jaclyn-taroni. I incorporated your wording suggestions and then also added a link to the metadata section of the downloads for the bullet about added metadata included in the download. This should be ready for another look.
